proto/celestia/core/v1/blob/blob.proto

```diff
@@ -21,4 +21,12 @@ message BlobTx {
   bytes tx = 1;
   repeated Blob blobs = 2;
   string type_id = 3;
-}
\ No newline at end of file
+}
+
+
+// BlobJson define a simple file in JSON format for user to pass in as argument 
+// in payForBlob CLI. 
+message BlobJson {
+  string namespace_id = 1;
+  string blob = 2;
+}
```

Comment:
adding this is understandable, but since the type is only internal then we should only define it as a go struct

Reply:
removed as requested